pkg/minibroker/minibroker.go

```diff
@@ -570,20 +536,20 @@ func (c *Client) bindSynchronously(
 			Services(releaseNamespace).
 			List(ctx, filterByInstance)
 		if err != nil {
-			return err
+			return errors.Wrapf(err, "failed to get services")
 		}
 		if len(services.Items) == 0 {
-			return osb.HTTPStatusCodeError{StatusCode: http.StatusNotFound}
+			return fmt.Errorf("failed to get services: no services found")
```

Comment:
I thought we needed to return the OSB-specific errors because the framework handle them specially?